fastestimator/pipeline.py

```diff
@@ -184,11 +186,14 @@ def get_loader(self, mode: str, epoch: int = 0,
             batch_size = self.batch_size
             if isinstance(batch_size, Scheduler):
                 batch_size = batch_size.get_current_value(epoch)
+            if isinstance(data, BatchDataset) and not batch_size is None:
+                raise ValueError("batch_size must be None when using BatchDataset")
```

Comment:
don't we want to just ignore the batch size and raise a warning? like if they use a scheduler and have non-batch dataset epoch 0-5 and then batch dataset after that

Reply:
this batch_size is the batch_size on that specific epoch, it won't be a scheduler.

Reply:
so what happens right now if I pass batch_size = 20 but use a scheduler that contains both a tf.dataset and an fe.dataset?

Reply:
we will raise a warning saying batch_size will only be used for built-in dataset